src/Google/GlobalSiteTag.php

```diff
@@ -78,6 +78,13 @@ function() use ( $ads_conversion_id, $ads_conversion_label ) {
 			},
 			1000000
 		);
+		add_action(
+			'wp_body_open',
+			function() {
+				$this->display_view_item_event_snippet();
+			},
+			1000002
```

Comment:
We can leave out the priority for this hook, in the previous ones it's done because we want it to be added to the end of wp_head, although here we use a different hook, so we can use the default priority.